Mathlib/Algebra/Lie/Nilpotent.lean

```diff
@@ -88,6 +88,41 @@ theorem lowerCentralSeries_succ :
     lowerCentralSeries R L M (k + 1) = ⁅(⊤ : LieIdeal R L), lowerCentralSeries R L M k⁆ :=
   (⊤ : LieSubmodule R L M).lcs_succ k
 
+theorem lowerCentralSeries_scalars_aux (R₁ R₂ L M : Type*)
+    [CommRing R₁] [CommRing R₂] [AddCommGroup M]
+    [LieRing L] [LieAlgebra R₁ L] [LieAlgebra R₂ L] [Module R₁ M] [Module R₂ M] [LieRingModule L M]
+    [LieModule R₁ L M] (k : ℕ) :
+    let I := lowerCentralSeries R₂ L M k; let S : Set M := {⁅a, b⁆ | (a : L) (b ∈ I)}
+    (Submodule.span R₁ S : Set M) ≤ (Submodule.span R₂ S : Set M) := by
+  intro I S x hx
+  simp only [SetLike.mem_coe] at hx ⊢
+  induction hx using Submodule.closure_induction with
+  | zero => exact Submodule.zero_mem _
+  | add y z hy₁ hz₁ hy₂ hz₂ => exact Submodule.add_mem _ hy₂ hz₂
+  | smul_mem c y hy =>
+      obtain ⟨a, b, hb, rfl⟩ := hy
+      rw [← smul_lie]
+      exact Submodule.subset_span ⟨c • a, b, hb, rfl⟩
+
+theorem lowerCentralSeries_scalars (R₁ R₂ L M : Type*) [CommRing R₁] [CommRing R₂] [AddCommGroup M]
+    [LieRing L] [LieAlgebra R₁ L] [LieAlgebra R₂ L] [Module R₁ M] [Module R₂ M] [LieRingModule L M]
+    [LieModule R₁ L M] [LieModule R₂ L M] (k : ℕ) :
+    (lowerCentralSeries R₁ L M k : Set M) = (lowerCentralSeries R₂ L M k : Set M) := by
```

Comment:
How about instead:
```suggestion
theorem coe_lowerCentralSeries_eq_int [LieModule R L M] (k : ℕ) :
    (lowerCentralSeries R L M k : Set M) = (lowerCentralSeries ℤ L M k : Set M) := by
  sorry
```
After all, the what `lowerCentralSeries_scalars` is really saying is that the `lowerCentralSeries` is also just a property of the Lie ring / Lie ring module.